lua/lspsaga/highlight.lua

```diff
@@ -14,6 +14,7 @@ local function hi_define()
     SagaSpinner = { link = 'Statement' },
     SagaText = { link = 'Comment' },
     SagaSelect = { link = 'String' },
+    SagaSep = { fg = 'white', bg = 'none' },
```

Comment:
I think Neovim defaults to the `Normal` when this one was not found. maybe better default needs to be defined, please feel free to change it